.github/workflows/upload-to-pypi.yml

```diff
@@ -0,0 +1,106 @@
+name: Upload-ansible-release-artifacts-to-pypi
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example : 11.1.0
+        required: true
+      ansible-major-version:
+        description: 'Example 11'
+        required: true
+
+env:
+  BUILD_DATA_DIR: "antsibull/build/ansible-build-data/${{ inputs.ansible-major-version }}"
+  BUILDFILE: "ansible-${{ inputs.ansible-major-version }}.build"
+  DEPSFILE: "ansible-${{ inputs.ansible-version }}.deps"
+
+# Build job builds the wheel and the source tarball and upload the artifacts
+
+jobs:
+  build:
+    name: Build sdist and wheel
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out antsibull
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-community/antsibull
+          path: antsibull
+
+      - name: Pre-create build directory
+        run: |
+          mkdir -p antsibull/build
+
+      - name: Check out ansible-build-data
+        uses: actions/checkout@v3
+        with:
+          path: antsibull/build/ansible-build-data
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        working-directory: antsibull
+        run: |
+          python3 -m pip install ansible-core antsibull
+          ansible-galaxy collection install community.general
+
+      - name: Create sdist and wheel using the deps file created in the ansible-build-data Release PR
+        working-directory: antsibull
+        run: |
+          ansible-playbook -vv playbooks/build-single-release.yaml -e 'antsibull_ansible-version=${{ inputs.ansible-version }}' -e antsibull_data_reset=false
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: sdist-and-wheel
+          path: antsibull/build/ansible-*.*
+
+# publish job downloads the arifacts and publish it to PyPI
+
+  publish:
+    needs: build
+    name: 'Upload Ansible (${{ inputs.ansible-version }}) to PyPI'
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/project/ansible/${{ inputs.ansible-version }}
```

Comment:
Is this the eight environment URL?

Reply:
@gotmax23 what do you mean? The link looks correct to me.